src/apis/weather-api.js

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+import { Context } from "../contex.js";
+
+const key = "KEY";
+const instance = axios.create({
+  baseURL: "https://api.openweathermap.org/",
```

Comment:
Better we create a `BASE_URL` variable above axios instance creation and set it to `baseUrl`.
Example:
`baseUrl: BASE_URL`